### PR TITLE
Added 200_60 and 200_85 quorums with extended phases

### PR DESCRIPTION
**Pull request by CodeIsTheKey**
Includes accelerated update vote for testnet starting 1440 blocks after current votes are locked in.

### CODE REVIEW

File: src/chainparams.cpp [hunk 1]
```diff
@@ -371,23 +371,32 @@ class CTestNetParams : public CChainParams {
...
+                false,                    // forceUpdate
+                VoteThreshold(85, 85, 1), // minerThreshold
+                VoteThreshold(0, 0, 1))   // nodeThreshold
+        );
+        updateManager.Add(
+            Update(EUpdate::QUORUMS_200_8, std::string("Quorums 200/8"),
+                2,                        // bit
+                1440,                     // roundSize
+                79200,                    // startHeight
+                3,                        // votingPeriod
```
Comment (CodeIsTheKey) [inline src/chainparams.cpp:394]: ~3 days average over 85%.  Once locked in, ~2 days until activation.

File: src/update/update.cpp [hunk 2]
```diff
@@ -82,8 +82,8 @@ VoteResult MinerRoundVoting::GetVote(const CBlockIndex *blockIndex, const Update
 
     // assert(height % update.RoundSize == 0);
     if (blockIndex->nHeight % update.RoundSize() != 0) {
-        LogPrint(BCLog::UPDATES, "Updates: MinerRoundVoting::GetVote, Height: %7d, Partial rounds not accepted\n",
-                 blockIndex->nHeight);
+        LogPrint(BCLog::UPDATES, "Updates: Update: %s, MinerRoundVoting::GetVote, Height: %7d, Partial rounds not accepted\n",
```
Comment (CodeIsTheKey) [inline src/update/update.cpp:85]: A few logs missing the name of the update.

File: src/evo/deterministicmns.cpp [hunk 3]
```diff
@@ -930,7 +930,7 @@ bool CDeterministicMNManager::BuildNewListFromBlock(const CBlock &block, const C
     }
 
     mnListRet = std::move(newList);
-    UpdateLLMQParams(mnListRet.GetAllMNsCount(), nHeight, sporkManager.IsSporkActive(SPORK_21_LOW_LLMQ_PARAMS));
+    UpdateLLMQParams(mnListRet.GetAllMNsCount(), pindexPrev, sporkManager.IsSporkActive(SPORK_21_LOW_LLMQ_PARAMS));
```
Comment (CodeIsTheKey) [inline src/evo/deterministicmns.cpp]: This previously passed in height, which was pindexPrev->nHeight + 1.  See any issues with using nHeight?

File: src/evo/deterministicmns.cpp [hunk 4]
```diff
@@ -1040,8 +1040,7 @@ CDeterministicMNList CDeterministicMNManager::GetListForBlock(const CBlockIndex
         }
     }
     // is this needed?
-    UpdateLLMQParams(snapshot.GetAllMNsCount(), snapshot.GetHeight(),
-                     sporkManager.IsSporkActive(SPORK_21_LOW_LLMQ_PARAMS));
+    UpdateLLMQParams(snapshot.GetAllMNsCount(), pindex, sporkManager.IsSporkActive(SPORK_21_LOW_LLMQ_PARAMS));
```
Comment (CodeIsTheKey) [inline src/evo/deterministicmns.cpp]: This should use the same height as before.

File: src/evo/deterministicmns.cpp (hunk 3 above)
Comment (CodeIsTheKey) [reply src/evo/deterministicmns.cpp]: I have updated this logic to use height + 1.

**Review by npq7721 — APPROVED** [review]
LGTM

**Pull request merged**